### PR TITLE
Add compat data for UserProximityEvent

### PR DESCRIPTION
**Pull request by maboa**
https://developer.mozilla.org/en-US/docs/Web/API/UserProximityEvent

### CODE REVIEW

File: api/UserProximityEvent.json [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+          },
+          "safari_ios": {
+            "version_added": false
+          },
+          "webview_android": {
+            "version_added": false
+          }
+        },
+        "status": {
+          "experimental": false,
```
Comment (ddbeck) [inline api/UserProximityEvent.json]: I think this is still experimental

**Comment by maboa** [comment]
Done. Thanks!

**Comment by ddbeck** [comment]
🎉

**Comment by connorshea** [comment]
I think this is missing `near`? https://developer.mozilla.org/en-US/docs/Web/API/UserProximityEvent/near

**Comment by ddbeck** [comment]
@connorshea Another good catch. Actually, there's a few issues with this data. I'll open another PR. I swear I'll get better at reviewing these 😄

**Pull request merged**